src/main/java/com/ncc/aif/AIFUtils.java

```diff
@@ -955,7 +955,7 @@ public static Resource makeHypothesis(Model model, String hypothesisURI, Set<Res
      * @param resource   The Resource to mark with the specified importance
      * @param importance The importance value with which to mark the specified Resource
      */
-    public static void markImportance(Resource resource, Integer importance) {
+    public static void markImportance(Resource resource, Double importance) {
```

Comment:
Not your issue, but while you're here, could you fix the copy-paste error in the JavaDoc for `markInformativeJustification` below?  importance --> informative justification.

Reply:
Fixed